### PR TITLE
Templates & static assets re-arranged to match views structure

### PR DESCRIPTION
**Pull request by niranjan94**
**What has been done here ?** 
- All static assets have been re-arranged to follow same pattern as views folder
- All templates have been re-arranged to follow same pattern as views folder
- Bower assets directory has been changed
- All references to files have been updated throughout
- All routes (except APIs) have a trailing slash at the end
- Javascript files have been cleaned up. (Missing semicolons, brackets, indentation etc etc)
- A few PEP errors have been resolved
- All `console.log` statements have been removed

Can be tested at https://vast-refuge-62115.herokuapp.com .

### CODE REVIEW

**Comment by niranjan94** [comment]
@SaptakS @shubham-padia @magdalenesuo this is a big refactor. So, I'll need some help reviewing this one ... :)

File: app/templates/gentelella/users/mytickets/mytickets_list.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
-{% extends 'gentelella/admin/base.html' %}
+{% extends 'gentelella/base.html' %}
 {% block title %}
     {{ page_content.title }}
 {% endblock %}
 {% block head_css %}
     {{ super() }}
-    <link rel="stylesheet" href="{{ url_for('static', filename='css/admin/explore.css') }}">
-    <link href="{{ url_for('static', filename='css/admin/mytickets.css') }}" rel="stylesheet">
+    <link rel="stylesheet" href="{{ url_for('static', filename='css/events/explore.css') }}">
```
Comment (shubham-padia) [inline app/templates/gentelella/users/mytickets/mytickets_list.html]: This file doesn't exist. Maybe this url should be changed to `css/guest/explore/explore.css`

Comment (niranjan94) [inline app/templates/gentelella/users/mytickets/mytickets_list.html]: @shubham-padia thanks :)

Comment (niranjan94) [inline app/templates/gentelella/users/mytickets/mytickets_list.html]: Fixed :white_check_mark:

**Comment by niranjan94** [comment]
@mariobehling @SaptakS @magdalenesuo @shubham-padia thanks a lot for your reviews :smile: .. I'll do a one last skim through everything and merge this ... _fingers crossed_

**Pull request merged**